packages/astro/src/core/errors/errors-data.ts

```diff
@@ -611,6 +611,25 @@ export const PrerenderDynamicEndpointPathCollide = {
 	hint: (filename: string) =>
 		`Rename \`${filename}\` to \`${filename.replace(/\.(?:js|ts)/, (m) => `.json` + m)}\``,
 } satisfies ErrorData;
+
+/**
```

Comment:
Noting that the location of the previous error shows up in the middle of a bunch of image errors? Maybe *both* of these belong in a different spot? 

<img width="680" height="458" alt="image" src="https://github.com/user-attachments/assets/73c4f82f-3f83-4782-bb45-1d602dd4b8e4" />